I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 242, in script
    barista "Hiya! How can I help you?"
Exception: Sayer 'barista' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 242, in script
    barista "Hiya! How can I help you?"
  File "/Users/cdean248345/renpy-7.4.11-sdk/renpy/ast.py", line 681, in execute
    who = eval_who(self.who, self.who_fast)
  File "/Users/cdean248345/renpy-7.4.11-sdk/renpy/ast.py", line 574, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'barista' is not defined.

Darwin-21.4.0-x86_64-i386-64bit
Ren'Py 7.4.11.2266
Zombie or Not 1.0
Mon May  9 21:51:47 2022
